cfg/cis-1.5/etcd.yaml

```diff
@@ -0,0 +1,131 @@
+---
+controls:
+version: 1.15
+id: 2
+text: "etcd"
+type: "master"
+groups:
+- id: 2
+  text: ""
```

Comment:
Should there be some text here?

Reply:
That would be my guess too. I'll see if I can find something suitable to add there.